src/main/java/pl/touk/sputnik/Test.java

```diff
@@ -5,10 +5,12 @@
     public boolean test = false;
 
     public int method() {
+        String str = null;
         if (test) {
-            return 1;
+            return str.length( ) + 4;
```

Comment:
\[FindBugs\] WARNING: NP: Null pointer dereference of str in pl\.touk\.sputnik\.Test\.method\(\)